search.go

```diff
@@ -48,6 +48,15 @@ type SearchCriteria struct {
 	Unseen     bool
 }
 
+func maybeString(mystery interface{}) string {
```

Comment:
Seems reasonable in this particular case because errors are ignored anyway when parsing fields.